docs/pages/admin-guides/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -176,7 +176,7 @@ teleport:
   # or the Auth Service address
   auth_server: 10.1.1.10:3025
 
-# Enable the SSH Service and disable the Auth and Proxy Services:
+# Enable the SSH Service and disable the Auth Service and Proxy Service Services:
```

Comment:
This sentence seemed to be correct before the change. Was it an erroneous search & replace or is Vale autofixing this somewhat too aggressively?

Reply:
This should be "Auth Service and Proxy Service", rather than "Auth Service and Proxy Service Services". The linter recommends "Auth Service and Proxy Service" because these are two separate proper names.

Reply:
Corrected